src/common.ts

```diff
@@ -1,2 +1,2 @@
-export const URLRegex = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}(\.[a-zA-Z0-9()]{1,6})?\b([-a-zA-Z0-9()@:,%_\+.~#?&//=]*)/gi;
```

Comment:
Why?

Reply:
I remember that ESint might have some warning about it -- however I am not sure whether the warning makes sense. We need to escape `+` in regex right?

Reply:
IDE auto fix, I don't know

Reply:
Rollback this line